src/common/transformations/src/transformations/common_optimizations/activations_scaling.cpp

```diff
@@ -0,0 +1,511 @@
+        }
+
+        if (has_bias) {
+            auto add = child_node->shared_from_this();
+            target_inputs = add->get_output_target_inputs(0);
+            auto scale_down_bias = std::make_shared<ov::op::v1::Multiply>(
+                add->input(bias_index).get_source_output(),
+                std::make_shared<ov::op::v0::Constant>(add->input(bias_index).get_element_type(),
+                                                       scale_shape,
+                                                       scale_down_value));
+            scale_down_bias->set_friendly_name(add->get_friendly_name() + "_scale_down");
+            ov::copy_runtime_info(add, scale_down_bias);
+            if (scale_down_bias->output(0).get_element_type() != scaled_prec && !keep_precision) {
+                auto convert_bias_prec = std::make_shared<ov::op::v0::Convert>(scale_down_bias->output(0), scaled_prec);
+                add->input(bias_index).replace_source_output(convert_bias_prec->output(0));
+            } else {
+                add->input(bias_index).replace_source_output(scale_down_bias->output(0));
+            }
+            add->revalidate_and_infer_types();
+            if (add->output(0).get_element_type() != output_prec && !keep_precision) {
+                output_of_scaled_op = std::make_shared<ov::op::v0::Convert>(add->output(0), output_prec);
+            } else {
+                output_of_scaled_op = add;
+            }
+        } else {
+            target_inputs = output_of_scaled_op->get_output_target_inputs(0);
+            if (output_of_scaled_op->output(0).get_element_type() != output_prec && !keep_precision) {
+                output_of_scaled_op =
+                    std::make_shared<ov::op::v0::Convert>(output_of_scaled_op->output(0), output_prec);
+            }
+        }
+
+        auto scale_up = register_new_node<ov::op::v1::Multiply>(
+            output_of_scaled_op->output(0),
+            std::make_shared<ov::op::v0::Constant>(output_of_scaled_op->output(0).get_element_type(),
+                                                   scale_shape,
+                                                   scale_up_value));
+        scale_up->set_friendly_name(scaled_op->get_friendly_name() + "_scale_up");
+        ov::copy_runtime_info(scaled_op, scale_up);
+        for (auto& in : target_inputs) {
+            in.replace_source_output(scale_up);
+        }
+
+        return true;
+    };
+
+    auto m = std::make_shared<ov::pass::pattern::Matcher>(scaled_op_m, "ScaleDownSingleLayer");
+    this->register_matcher(m, callback);
+}
+
+// ScaleDownFusion merges multiple scale_down layers into one.
+//
+//               input                      input
+//               /   \           ==>          |
+//           Mul_a   Mul_b                  Mul_a
+//             |       |                    /   \_
+//           op_a    op_b                op_a   op_b
+ov::pass::activations_scaling::ScaleDownFusion::ScaleDownFusion() {
```

Comment:
I believe we already have a transformation to fuse the shared from the same input ops 
Could you check if this transformation works for this case? Probably it's better to extend it if it doesn't

https://github.com/openvinotoolkit/openvino/blob/1ec91fc6b64def702aa42c3271ba6f97bb6ef22e/src/common/transformations/src/transformations/common_optimizations/shared_ops_optimization.cpp#L188